engines/bastion_katello/app/assets/javascripts/bastion_katello/errata/errata.controller.js

```diff
@@ -31,6 +31,10 @@ angular.module('Bastion.errata').controller('ErrataController',
         $scope.table = nutupane.table;
         $scope.removeRow = nutupane.removeRow;
 
+        Erratum.queryUnpaged({'organization_id': CurrentOrganization}, function (result) {
+            $scope.errataCount = result.total;
+        });
+
```

Comment:
I just realized I was using `queryUnpaged()`!  Let me try with `queryPaged()` that shouldn't slow it down as much.